chipsec/utilcmd/mmio_cmd.py

```diff
@@ -21,15 +21,21 @@
 """
 >>> chipsec_util mmio list
 >>> chipsec_util mmio dump <MMIO_BAR_name> [offset] [length]
+>>> chipsec_util mmio dump-abs <MMIO_base_address> <length>
```

Comment:
Can/should we include an offset in the dump-abs command? Seems like all the abs should match, either all have it or all don't?

Reply:
Sure, not a problem.  I will push a new patch.